### PR TITLE
Release 15.0.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "theforeman/foreman_proxy",
-      "version_requirement": ">= 13.0.0 < 17.0.0"
+      "version_requirement": ">= 13.0.0 < 18.0.0"
```
Comment (ekohl) [inline metadata.json:25]: I prefer these to at least be in separate commits. Releasing and allowing a new version are separate actions. You may even want to cherry pick allowing a newer version but you really don't cherry pick releases.

**Review by ekohl — COMMENTED** [review]
Changelog looks good.

**Comment by ehelms** [comment]
Split it out to its own commit

**Review by ekohl — APPROVED** [review]

**Review by ekohl — CHANGES_REQUESTED** [review]
Oh, I see some cleanup that should happen before. PR incoming.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Derp, didn't check, but you should probably rebase + regenerate to include my PR.
